easybuild/easyconfigs/w/WPS/WPS-4.1_netCDF-Fortran_seperate_path.patch

```diff
@@ -0,0 +1,85 @@
+# Allow netCDF library with separate directories for C and Fortran
+# ============================================================================
+# This patch has been around in EasyBuild since 2013; it was committed by
+# @boegel then.  Adapted by @andreas-h to accomodate WPSv4 and foss toolchain
+#
+# updated for WPS v4.1 yby Kenneth Hoste (HPC-UGent)
+
+--- WPS-4.1/arch/Config.pl.orig	2019-04-12 23:48:41.000000000 +0200
++++ WPS-4.1/arch/Config.pl	2019-12-14 15:27:30.931325333 +0100
+@@ -41,6 +41,7 @@
+     if(substr( $ARGV[0], 1, 8 ) eq "netcdff=")
+     {
+         $sw_netcdff_lib = substr( $ARGV[0], 9);
++        $sw_netcdff_lib =~ s/!/ /g ;
+     }
+     if(substr( $ARGV[0], 1, 6 ) eq "phdf5=")
+     {
+--- WPS-4.1/arch/preamble.orig	2019-04-12 23:48:41.000000000 +0200
++++ WPS-4.1/arch/preamble	2019-12-14 15:27:30.933363000 +0100
+@@ -39,7 +39,7 @@
+                         -I$(WRF_DIR)/external/io_grib1 \
+                         -I$(WRF_DIR)/external/io_int \
+                         -I$(WRF_DIR)/inc \
+-                        -I$(NETCDF)/include
++                        -I$(NETCDF)/include -I$(NETCDFF)/include
+ 
+ WRF_LIB         =       -L$(WRF_DIR)/external/io_grib1 -lio_grib1 \
+                         -L$(WRF_DIR)/external/io_grib_share -lio_grib_share \
+--- WPS-4.1/configure.orig	2019-04-12 23:48:41.000000000 +0200
++++ WPS-4.1/configure	2019-12-14 15:28:37.173438862 +0100
+@@ -119,9 +119,14 @@
+   echo "Will use NETCDF in dir: $NETCDF"
+   # for 3.6.2 and greater there might be a second library, libnetcdff.a .  Check for this and use
+   # if available
+-  NETCDFF=" "
+-  if [ -f "$NETCDF/lib/libnetcdff.a" ] ; then
+-    NETCDFF="-lnetcdff"
++  USENETCDFF=" "
++  # for netCDF 4.2 and greater, the Fortran library is a seperate install
++  if [ -z "$NETCDFF" ]
++  then
++    NETCDFF=$NETCDF
++  fi
++  if [ -f "$NETCDFF/lib/libnetcdff.a" ] ; then
++    USENETCDFF="-L$NETCDFF/lib!-lnetcdff"
```

Comment:
@boegel I guess this should be:
```
USENETCDFF="-L$NETCDFF/lib -lnetcdff"
```

Reply:
No, there's a reason for this I think... The `!` is replaced with a space later somewhere.

It's also like this in the original patch `WPS_netCDF-Fortran_seperate_path.patch`